javascript/src/seedsource-ui/containers/RegionStep.tsx

```diff
@@ -25,6 +25,17 @@ type RegionStepProps = ConnectedProps<typeof connector> & {
   number: number
 }
 
+const helpTooltip = (
+  <p>
+    {t`The map is broken into large geographic regions such as the Western US and the Eastern US.`}&nbsp;
+    {t`Select`} <strong>{t`Automatic`} </strong>
+    {t`if your seedlot(s) and planting site(s) are from the same region (most common).`}&nbsp;
+    {t`Select`} <strong>{t`Custom`} </strong>
+    {t`if they are from different regions, and then specify the region that does not include the location chosen 
+    in step 2.`}
```

Comment:
It's generally best to keep localized sentences complete. Translation may involve reordering words, and may depend on context. For example, "select" can be a verb or an adjective (noun too), and the Spanish word is different for each: seleccionar (_seleccionas una manzana_ / _select an apple_) vs selecta (_una manzana selecta_ / _a select apple_). Context is also important in Spanish for determining correct gender (selecto vs selecta) and verb conjucation.

I generally accomplish this by creating variables for open and close tags, then using them in the localization:

```
t{`Select {openTag}Custom{closeTag} if they are ...etc`}
```

Reply:
I'll update. thanks for the tip.

Reply:
I tried this:
`t{`Select ${openTag}Custom${closeTag} if they are ...etc`}`

which resulted in plain text tags.

I'm looking into rendering the html tags as html, but I'm coming up empty

Reply:
@nikmolnar Could you take a look at what I ended up doing here? I'm not sure if our translations are considered a trusted source as I ended up having to parse the whole string as html.

Reply:
Gotcha. That makes more sense now. So the individual bold words can be translated as a piece and then re-arranged as necessary in the larger paragraph when translated.